### ISSUE TITLE
Cannot filter text like parallel-n64 core

### ISSUE THREAD

**Issue by nelsongt** [libretro/mupen64plus-libretro-nx#325]
Hello thank you for your work, I recently switched to this core for the advertised superior performance on my x86 HTPC running windows10. I am using parallel rdp and rsp plugins. Disclaimer - Not sure if this is a bug or not.

On parallel-n64 core I used before the non-vectorized or bitmapped text in games like Diddy Kong Racing was heavily blurred/anti-aliased/filtered to smooth the pixelated edges. I tried to match all core settings as best as possible from old core to this one but in this core I cannot achieve same effect. Personal tastes as to what looks better aside, is it possible to filter the text like the old core? I can upload screenshots if needed. I thought maybe the text (sprites?) I refer to is a framebuffer element so I tried to enable SSAA on framebuffer but this did not appear to change anything in Diddy Kong Racing. Thanks

**Comment by m4xw** [comment]
Not sure what you are talking about

**Comment by nelsongt** [comment]
Yes I do have the familiarity to say the right words. I added screenshots ^

**Comment by m4xw** [comment]
That just looks like bilinear to me.
Did you try playing with the hybrid filter?

**Comment by m4xw** [comment]
Ah forgot hybrid was gliden
So talked to maister, might be force native tex_rect.
I vaguely remember i forgot to add that one, so will investigate

**Comment by m4xw** [comment]
Actually seems i added it https://github.com/libretro/mupen64plus-libretro-nx/blob/develop/libretro/libretro_core_options.h#L987-L993

Did u try?

**Comment by nelsongt** [comment]
Sorry, I operate on a slow time scale to most people. Yes I looked for the hybrid filter but only saw the option under GlideN64 plugin. I did try the native tex_rect option I think multiple times but did not see any difference. Does this require restarting the core? I can test again tonight.

**Comment by nelsongt** [comment]
Was not able to get those texture objects to blur any differently on 2 different Windows 10 PCs and one Linux.
